ci-operator/jobs/openshift/release/openshift-release-release-4.8-periodics.yaml

```diff
@@ -5132,6 +5132,109 @@ periodics:
     - name: release-pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cluster: build02
+  cron: '@yearly'
+  decorate: true
+  labels:
+    job-env: aws
+    job-release: "4.8"
+    job-test: e2e
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: release-openshift-origin-installer-e2e-aws-ovn-upgrade-rollback-4.7-to-4.8
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --secret-dir=/usr/local/pull-secret
+      - --secret-dir=/usr/local/e2e-$(CLUSTER_TYPE)-upgrade-cluster-profile
+      - --target=e2e-$(CLUSTER_TYPE)-upgrade
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+      - --gcs-upload-secret=/secrets/gcs/service-account.json
+      - --lease-server-credentials-file=/etc/boskos/credentials
+      command:
+      - ci-operator
+      env:
+      - name: RELEASE_IMAGE_INITIAL
+      - name: RELEASE_IMAGE_LATEST
+      - name: CLUSTER_TYPE
+        value: aws
+      - name: UNRESOLVED_CONFIG
```

Comment:
There should be no reason to add a handcrafted job now. The same result should be achieved by adding the test item stanza to the appropriate ci-op config (I think it will be this one https://github.com/openshift/release/blob/master/ci-operator/config/openshift/release/openshift-release-master__ci-4.8-upgrade-from-stable-4.7.yaml ) and run `make jobs`.

Reply:
Thanks for the help @petr-muller. See if this current version looks right please :)